scripts/install-tuist.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e
+
+shell_join() {
+  local arg
+  printf "%s" "$1"
+  shift
+  for arg in "$@"; do
+    printf " "
+    printf "%s" "${arg// /\ }"
+  done
+}
+
+ohai() {
+  printf "${tty_blue}==>${tty_bold} %s${tty_reset}\n" "$(shell_join "$@")"
+}
+
+warn() {
+  printf "${tty_red}Warning${tty_reset}: %s\n" "$(chomp "$1")"
+}
+
+# The line below extracts the version from the local .tuist-version file and installs that version tag from the github releases.
+LATEST_VERSION=$(cat .tuist-version)
```

Comment:
The installed Tuist version will always match what we've pinned it to.

Reply:
It's worth noting that this installation never actually installs Tuist itself but rather tuistenv which was their prior version manager, and what they're moving away from in favour of mise. If you look at the terminal output the actual Tuist version is installed when you issue the first command, such as `generate`.